src/locales/en.json

```diff
@@ -45,7 +45,8 @@
       "BTC": "Bitcoin address",
       "ORDINALS": "Ordinals address",
       "STX": "Stacks address",
-      "ALERT": "Verify that these addresses match the addresses displayed on the device."
+      "ALERT": "Verify that these addresses match the addresses displayed on the device.",
+      "ALREADY_CONNECTED_WARNING": "Ledger device has already been connected to your Xverse wallet. Would you like to create another account?"
```

Comment:
An account has already been added to your wallet from this device. Would you like to create a new account?